src/utils/string_utils.h

```diff
@@ -45,15 +45,18 @@ inline std::string GetCString(v8::Isolate* isolate, v8::Local<v8::Value> val) {
 inline int GetLength(v8::Isolate* isolate, v8::Local<v8::Value> val) {
     if (val->IsString()) {
         return v8::String::Cast(*val)->Length();
+    } else if (val->IsStringObject()) {
+        return v8::StringObject::Cast(*val)->ValueOf()->Length();
     } else {
         if (val->IsUndefined()) {
             return 0;
         } else if (val->IsNull()) {
             return 0;
         } else {
-            auto context = isolate->GetCurrentContext();
-            auto firstLocalString =  val->ToString(context).ToLocalChecked();
-            return firstLocalString->Length();
+            return v8::String::kMaxLength;
```

Comment:
Before `GetLength` received only strings so I suppose this was never the case.
But now a `Buffer` would fit in this case and I think we should avoid obtaining the string representation of the Buffer because it could change after we taint it and because it could be an expensive operation.
So just return the v8 max string length.